stix_shifter_modules/onelogin/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,61 @@
+{
+  "user-account": {
+    "fields": {
+      "user_id": ["user_id"],
+      "display_name": ["user_name"]
+    }
+  },
+  "ipv4-addr": {
+    "fields": {
+      "value": ["ipaddr"]
+    }
+  },
+
+  "x-onelogin-finding": {
```

Comment:
Just make this `x-onelogin` to follow the convention of other connector's custom attributes.

Reply:
Actually, ignore that suggestion, though I think some of these may fix in the `x-ibm-finding` object, such as policy_name.

Reply:
Done @delliott90